server/log.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+/**
+ * redis.js
+ *
+ * Used to store Audit logs.
+ */
+
+const Redis = require("redis");
+const logLength = 5000;
```

Comment:
Should this be an environment variable? If it was it would be easier to update on Heroku if it was. /cc @railsagainstignorance